torch/_lazy/computation.py

```diff
@@ -0,0 +1,15 @@
+import torch._C._lazy
```

Comment:
it seems a bit arbitrary that these are grouped together under `computation`.  For example, why didn't `sync_multi` make the cut into this file?

Reply:
yes, I don't find a hard rule to decide which APIs goes to which module. This file specifically contains APIs needed to reused a compiled graph so far.  `sync_multi` is a more general API thus I didn't add that here. But it should also look good to put `sync_multi` here